tests/test_client.py

```diff
@@ -705,8 +702,8 @@ def test_get_adc_env_from_well_known(self, *stubs):
         # Make sure the well-known file actually doesn't exist.
         self.assertTrue(os.path.exists(get_well_known.return_value))
 
-        method_name = \
-            'oauth2client.client._get_application_default_credential_from_file'
+        method_name = ('oauth2client.client.'
```

Comment:
I assume you mean break == line break == newline.

Going with the general philosophy that multiline continuations should just jump to the 2nd line immediately?

I folded the change into the original commit